src/components/Cluster/NewCluster/CreateNodePoolsCluster.js

```diff
@@ -280,19 +280,23 @@ class CreateNodePoolsCluster extends Component {
       np => np.data
     );
 
+    let createPayload = {
+      owner: this.props.selectedOrganization,
+      name: this.state.name.value,
+      release_version: this.props.selectedRelease,
+    };
+
+    if (this.state.hasAZLabels) {
+      // TODO: don't use array here as long as there can be only one master node.
```

Comment:
I leave this TODO comment here instead of solving it, as I don't want to make the fix bigger than it is.